client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-content/style.scss

```diff
@@ -0,0 +1,10 @@
+
+@import '@wordpress/base-styles/_breakpoints.scss';
+@import '@wordpress/base-styles/_mixins.scss';
+
+.site-content {
+	&__no-sites {
+		text-align: center;
+		font-size: 1.5rem;
+	}
+}
```

Comment:
This comment comes from a personal preference (I don't know if it is part of our conventions), but for searchability purposes, it's better to write the full name of a class rather than using the `&` shortcut.  In the former case, searching for `site-content__no-sites` leads you straight to the file where the class definition lives.

Reply:
I see we are using this in most of the places. I don't remember seeing this as a part of our convention anywhere. But, I agree that this will be better for searchability.

Reply:
Most often the SCSS `&` is used with the `classNames` when classes are added based on boolean values. The pattern looks something like this:

```
// In the React code
<Component className={ "main-class", { optional-class: isOptionalClass() }/>

// In the SCSS
.main-class {
    // Styles for the main-class
    &.optional-class {
        // Styles specific only to the optional class
    }
}
```

This is a good way to alter styles based on things calculated in code while keeping the class names searchable.